aws-scenario2/private-resources.tf

```diff
@@ -16,7 +16,7 @@ resource "aws_security_group" "db_sg" {
     from_port   = 0
     to_port     = 0
     protocol    = "-1"
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = [var.egress_cidr]
   }
```

Comment:
@coderabbitai build test file for this archive

Reply:
how to add this test on the repository and how can i run it?

Reply:
@coderabbitai 

how to add this test on the repository and how can i run it?

Reply:
is it a good practice to have a test written in go in the same repository written in terraform?

Reply:
generate interesting stats about this repository and render them as a table.

Reply:
@coderabbitai fale algo